src/stock-items/add-stock-item/stock-item-units-edit/stock-item-units-edit.component.tsx

```diff
@@ -120,7 +120,10 @@ const StockItemUnitsEdit: React.FC<StockItemUnitsEditProps> = ({
         name="purchasePriceUoMUuid"
         controllerName="purchasePriceUoMUuid"
         control={control}
-        title={t("purchasePricePackagingUnit", "Purchase price packaging unit")}
+        title={t(
+          "purchasePricePackagingUnit",
```

Comment:
Do you know what this is supposed to mean?

Reply:
The title of that field